### PR TITLE
EL-1453 Replace gulp with django-webpack-loader

### PR DESCRIPTION
**Pull request by starswan**
## What does this pull request do?

Replace gulp with django-webpack-loader

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [ ] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: fala/assets-src/sass/style.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
-// Set up routing so requests for files in /static are served from /node_modules/govuk-frontend/dist/govuk/all.
 // import GDS components from `node_modules`
-$govuk-images-path: "/static/images/";
-$govuk-fonts-path: "/static/fonts/";
+//$govuk-images-path: "/assets/images/";
+//$govuk-fonts-path: "/assets/fonts/";
```
Comment (MazOneTwoOne) [inline fala/assets-src/sass/style.scss:3]: @starswan might want to get rid of these if not used.

File: fala/settings/base.py [hunk 2]
```diff
@@ -90,6 +104,10 @@ def root(*x):
         "DIRS": [root("templates"), abspath(root(project_root, "node_modules", "mojular-templates"))],
```
Comment (MazOneTwoOne) [inline fala/settings/base.py:104]: @starswan we can rid of `mojular-templates`, I think

File: fala/settings/base.py [hunk 3]
```diff
@@ -184,7 +202,13 @@ def root(*x):
 # Python dotted path to the WSGI application used by Django's runserver.
 WSGI_APPLICATION = "fala.wsgi.application"
 
-INSTALLED_APPS = ("django.contrib.sessions", "django.contrib.messages", "django.contrib.staticfiles", "requests")
+INSTALLED_APPS = (
```
Comment (MazOneTwoOne) [inline fala/settings/base.py:205]: @starswan I think this can be in square brackets

Comment (starswan) [reply fala/settings/base.py:205]: I think its more idiomatic for this to be a tuple - lists are mutable, this is not.

**Pull request merged**